codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/Hearth.java

```diff
@@ -82,6 +82,11 @@ public final class Hearth {
             .name("JSON")
             .build();
 
+    public static final Symbol CBOR = Symbol.builder()
+            .namespace("Hearth", "::")
+            .name("Cbor")
```

Comment:
I think the class/namespace should be CBOR technically. We did the pascal case in v3 because that was convention there. But here we've done JSON, XML, and now CBOR.

Reply:
Ah yeah, I think that makes sense.  Will move it.